input/MergeStars/ics.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
```

Comment:
Add a preamble which describes the purpose of this script and how it should be run.

Reply:
I included some information in the README about all the scripts, do you think it needs to be here as well? You can also get information by running "python ics.py -h"... maybe I should include that in the README.

Reply:
I think you still need some comments at the top of each of the .py scripts